packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/factories/standard-index.factory.ts

```diff
@@ -102,7 +102,9 @@ export class StandardIndexFactory {
           const hasAllFields = workspaceIndexMetadataArgs.columns.every(
             (expectedField) => {
               return objectMetadata.fields.some(
-                (field) => field.name === expectedField,
```

Comment:
based on TODO comment, I feel we should rather remove the filter?

Reply:
yeah of course !

Reply:
silly me